chips/litex/src/liteeth.rs

```diff
@@ -155,6 +155,7 @@ impl<const MAX_TX_SLOTS: usize, R: LiteXSoCRegisterConfiguration> LiteEth<'_, MA
         self.initialized.set(true);
     }
 
+    #[allow(clippy::mut_from_ref)]
```

Comment:
@lschuermann This function is lacking any safety documentation, do you have any concern with this allow?